scripts/inc.compatibility.sh

```diff
@@ -509,9 +509,11 @@ photosvrmode"
 }
 
 api25hack(){
-  if [ "$API" -ge "25" ]; then
+  if [ "$API" -eq "25" ]; then
     gappscore="$gappscore
-gmssetup"
```

Comment:
@SpasilliumNexus, I assume that that the removal of 'gmssetup' here should be added in api26hack() and api27hack() below?

Reply:
I am going to merge this and the "all" one.  I'll fix this when I hear from you (or not if it is correct).  I have one other change to opengapps/opengapps to make as well

Reply:
Commited: https://github.com/opengapps/opengapps/commit/c843fe13f77b5a19cdd2a91bec65ff54280f171e

Reply:
Hey sorry about not getting back. I didn't have Gmail set up for notifications on my "dev" account after a clean ROM install >_<

But yes, that's where I mean to add it at. Thank you for the fix!